rest-api/model/participant_summary.py

```diff
@@ -241,3 +241,4 @@ def siteId(cls):
       ParticipantSummary.numBaselineSamplesArrived)
 Index('participant_summary_hpo_withdrawal_status_time', ParticipantSummary.hpoId,
       ParticipantSummary.withdrawalStatus, ParticipantSummary.withdrawalTime)
+Index('participant_summary_last_modified', ParticipantSummary.lastModified)
```

Comment:
I think you need to run generate_schema -- there needs to be an alembic migration generated in order for this to do anything.

We might want to consider instead making this index on awardee, lastModified... expected usage is always syncing data for a single awardee.

Reply:
added hpo_id, still need to run migration and push it up

Reply:
ok done now